### PR TITLE
Support Windows/UWP SDK on ARM x64 architecture

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [x] (Waiting for #1279) Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Support Windows/UWP SDK on ARM x64 architecture

## Related PRs or issues

[AB#72178](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/72178)

### CODE REVIEW

File: Apps/Contoso.UWP.Demo/MainPage.xaml.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
```
Comment (guperrot) [inline Apps/Contoso.UWP.Demo/MainPage.xaml.cs]: What is this change?

File: Apps/Contoso.UWP.Puppet/MainPage.xaml.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
```
Comment (guperrot) [inline Apps/Contoso.UWP.Puppet/MainPage.xaml.cs]: What is this change?

File: Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs [hunk 3]
```diff
@@ -17,6 +21,18 @@ public class AppCenterTest
         [TestInitialize]
         public void InitializeAppCenterTest()
         {
+            // Mock the channel and group channel.
+            Mock<IChannelUnit> mockChannel = new Mock<IChannelUnit>();
```
Comment (guperrot) [inline Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs]: Use var when possible

File: Apps/Contoso.UWP.Demo/MainPage.xaml.cs (hunk 1 above)
Comment (MatkovIvan) [reply Apps/Contoso.UWP.Demo/MainPage.xaml.cs]: Resolving name conflict after target update. Was already approved in #1273

File: Apps/Contoso.UWP.Puppet/MainPage.xaml.cs (hunk 2 above)
Comment (MatkovIvan) [reply Apps/Contoso.UWP.Puppet/MainPage.xaml.cs]: https://github.com/microsoft/appcenter-sdk-dotnet/pull/1278#discussion_r363648179

File: Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs (hunk 3 above)
Comment (MatkovIvan) [reply Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs]:
```suggestion
            var mockChannel = new Mock<IChannelUnit>();
```

File: Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs [hunk 4]
```diff
@@ -17,6 +21,18 @@ public class AppCenterTest
...
+            Mock<IChannelGroup> mockGroup = new Mock<IChannelGroup>();
```
Comment (MatkovIvan) [inline Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs]:
```suggestion
            var mockGroup = new Mock<IChannelGroup>();
```

File: Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs [hunk 5]
```diff
@@ -17,6 +21,18 @@ public class AppCenterTest
...
+            mockGroup.Setup(mock => mock.AddChannel(It.IsAny<string>(), It.IsAny<int>(), It.IsAny<TimeSpan>(), It.IsAny<int>())).Returns(mockChannel.Object);
+
+            // Mock the channel group factory.
+            Mock<IChannelGroupFactory> mockGroupFactory = new Mock<IChannelGroupFactory>();
```
Comment (MatkovIvan) [inline Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs]:
```suggestion
            var mockGroupFactory = new Mock<IChannelGroupFactory>();
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 2.6.5 （Under development）
 
+### App Center
+
+* **[Feature]** Support ARM64 architecture for UWP platform.
+* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF/WinForms projects and minimal OS version to 16299 for UWP projects.
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF/WinForms projects and minimal OS version to 16299 for UWP projects.
* **[Dependency changes]** Update `SQLitePCLRaw.bundle_green` dependency to version `2.0.2` and remove dependency from `sqlite-net-pcl`.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,12 @@
 
 ## Version 2.6.5 （Under development）
 
+### App Center
+
+* **[Feature]** Support ARM64 architecture for UWP platform.
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion

#### UWP

* **[Feature]** Support ARM64 architecture.

#### UWP/WPF/WinForms

```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,18 @@
...
+### App Center
+
+
+#### UWP
+
+* **[Feature]** Support ARM64 architecture.
+
+#### UWP/WPF/WinForms
+
+* **[Breaking change]** Bump minimal versions: .NET Framework to 4.6.1 for WPF/WinForms projects and minimal OS version to 16299 for UWP projects.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: > minimal OS version to 16299 for UWP projects.

it's in the wrong section now

**Pull request merged**